Data-Science/README.md

```diff
@@ -0,0 +1,148 @@
+# DATA SCIENCE Roadmap
+
+## Introduction
+
+### what is Data science
+
+- [what is data science? -ibm.com](https://www.ibm.com/topics/data-science)
+- [what is data science? -aws.amazon.com](https://aws.amazon.com/what-is/data-science/)
+- [what is data science? -simplilearn.com](https://www.simplilearn.com/tutorials/data-science-tutorial/what-is-data-science)
+
+### Programming Languages
+
+- [R - good tool for visualization and statistical analysis.]
+
+  - [R programming tutorial -Youtube(freeCodeCamp.org)](https://youtu.be/_V8eKsto3Ug)
+  - [R For Data Science Full Course -Youtube(Simplilearn)](https://youtu.be/iROHLA_TXQM)
+  - [R programming language Introduction -GFG(GeeksForGeeks)](https://www.geeksforgeeks.org/r-programming-language-introduction/)
+  - [R programming Tutorial -W3schools(W3schools.com)](https://www.w3schools.com/r/)
+  - [R programming language -R for datascience (Hadley Wickham)](https://r4ds.had.co.nz/)
+
+- [Python]
+
+  - [Python Tutorial for Beginners -Youtube(TechWorld with Nana)](https://youtu.be/t8pPdKYpowI)
+  - [Python Tutorial - Python Full Course for Beginners - Youtube(Programming with Mosh)](https://youtu.be/_uQrJ0TkZlc)
+  - [Harvard CS50’s Introduction to Programming with Python - Youtube(freeCodeCamp.org)](https://youtu.be/nLRL_NcnK-4)
+  - [Python for Beginners - Youtube(freeCodeCamp.org)](https://youtu.be/eWRfhZUzrAc)
+  - [Python programming language -W3Schools(W3Schools.com)](https://www.w3schools.com/python/python_intro.asp)
+  - [Python programming language -GFG(GeeksForGeeks)](https://www.geeksforgeeks.org/python-programming-language/)
+
+- [SQL]
+  - [SQL Tutorial - Full Course - Youtube(freeCodeCamp.org)](https://youtu.be/HXV3zeQKqGY)
+  - [SQL Tutorial for Beginners -Youtube(Programming With Mosh)](https://youtu.be/7S_tz1z_5bA)
+  - [SQL for Data Analysis -W3schools(W3schools.com)](https://www.w3schools.com/sql/)
+  - [SQL for Data Analysis -javatpoint(javatpoint.com)](https://www.javatpoint.com/sql-tutorial)
+
+### Pandas
+
+- [Pandas Tutorials -Youtube(Corey Schafer)](https://youtube.com/playlist?list=PL-osiE80TeTsWmV9i9c58mdDCSskIFdDS)
+- [Best practices with pandas -Youtube(Data School)](https://youtube.com/playlist?list=PL5-da3qGB5IBITZj_dYSFqnd_15JgqwA6)
+- [Pandas -Kaggle(Kaggle.com)](https://www.kaggle.com/learn/pandas)
+- [Pandas documentation -Pandas.pydata(Pandas.pydata.org)](https://pandas.pydata.org/pandas-docs/version/0.15/tutorials.html)
+
+### Numpy
```

Comment:
These two should come under a frameworks category imo

Reply:
yeah this will be great